source/docs/api-reference/mechanisms/swerve/swerve-builder-api.rst

```diff
@@ -185,11 +185,24 @@ Full Example
 
          // The stator current at which the wheels start to slip;
          // This needs to be tuned to your individual robot
-         private static final double kSlipCurrentA = 300.0;
+         private static final double kSlipCurrentA = 150.0;
+
+         // Initial configs for the drive and steer motors and the CANcoder; these cannot be null.
+         // Some configs will be overwritten; check the `with*InitialConfigs()` API documentation.
+         private static final TalonFXConfiguration driveInitialConfigs = new TalonFXConfiguration();
+         private static final TalonFXConfiguration steerInitialConfigs = new TalonFXConfiguration()
+            .withCurrentLimits(
+               new CurrentLimitsConfigs()
+                  .withStatorCurrentLimit(40)
```

Comment:
I would like a short comment on why we are using such a low stator current here. It's too easy for folks to copy gains without thinking about it.